src/get-started/install.md

```diff
@@ -68,9 +64,9 @@ or file an issue on [GitHub](https://github.com/exonum/exonum/issues).
 
 ## Adding environment variables
 
-This variables needed for linking with the `rocksdb` and `snappy` libraries.
+These variables are needed for linking with the `rocksdb` and `snappy` libraries.
```

Comment:
If your OS contains pre-compiled libraries `rocksdb` or `snappy`, you may setup environment variables. This significantly reduce a compile time.

Reply:
@SerhiiOryshych Could you please add this explanation to the document?